cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,89 @@
 /// <reference types='cypress' />
+import { generateWorker } from '../support/generate';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  const worker = generateWorker();
 
+  function addWorker() {
+    cy.get('#addNewRecordButton').click();
+
+    cy.get('#firstName').type(worker.firstName);
+    cy.get('#lastName').type(worker.lastName);
+    cy.get('#userEmail').type(worker.email);
+    cy.get('#age').type(worker.age);
+    cy.get('#salary').type(worker.salary);
+    cy.get('#department').type(worker.department);
+
+    cy.get('#submit').click();
+
+    return worker;
+  }
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should add, edit and delete new worker', () => {
+    addWorker();
+
+    cy.get('#searchBox').type(worker.firstName);
+    cy.get('#edit-record-4').click();
+    cy.get('#userEmail').clear();
+    cy.get('#userEmail').type(`new${worker.email}`);
+    cy.get('#submit').click();
+    cy.contains('div', `new${worker.email}`).should('be.visible');
+
+    cy.get('#delete-record-4').click();
+    cy.contains('div', `new${worker.email}`).should('not.exist');
+  });
+
+  it('should implement pagination', () => {
+    addWorker();
+    addWorker();
+    addWorker();
+    addWorker();
+
+    cy.get('select[aria-label="rows per page"]').select('5');
+    cy.contains('.-next', 'Next').click();
+    cy.contains('div', worker.firstName).should('be.visible');
```

Comment:
The test assumes that the worker's first name will be visible on the next page. This might not be reliable if the order of workers changes or if there are more than 5 workers.